maxminddb/CMakeLists.txt.maxminddb

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+project(maxminddb VERSION 0.1.0 LANGUAGES C)
+
+set(CPACK_PACKAGE_NAME luasandbox-${PROJECT_NAME})
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Lua MaxMindDB Module")
+set(CPACK_PACKAGE_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
+set(CPACK_PACKAGE_VENDOR        "Mozilla Services")
+set(CPACK_PACKAGE_CONTACT       "Mike Trinkala <trink@mozilla.com>")
```

Comment:
Hi @trink 
Sorry for the late reply.
Do you see an easy way to have such standalone project and not duplicate CMakeLists.txt?
Maybe a ```-DEXTERNAL_EXT_<ext-name>=<ext-path>```?